sdk/openai/openai/test/public/completions.spec.ts

```diff
@@ -65,9 +65,18 @@ describe("OpenAI", function () {
       });
 
       describe("streamCompletions", function () {
+        let modelName: string;
+
+        before(async function (this: Context) {
+          if (authMethod === "OpenAIKey") {
+            modelName = "gpt-3.5-turbo-instruct";
+          } else {
+            modelName = "gpt-35-turbo-instruct";
+          }
+        });
+
```

Comment:
Why is this needed?

Reply:
The "text-davinci-003" model the test used had been deprecated and this is the replacement model I use for the test.

Reply:
Could we move it to the matrix setup where no specific model is hard-coded?

Reply:
Sure. I'll update the `withDeployments` methods so that we can pass in the options for the assertion method